repository/build.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junitVersion
+    testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junitVersion
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: junitVersion
+    testImplementation group: 'com.h2database', name: 'h2', version: h2Version
+
+    implementation group: 'mysql', name: 'mysql-connector-java', version: mysqlConnectorVersion
+    implementation group: 'org.apache.commons', name: 'commons-dbcp2', version: dbcpVersion
+    implementation group: 'org.springframework', name: 'spring-jdbc', version: springVersion
+    implementation group: 'org.springframework', name: 'spring-context', version: springVersion

```

Comment:
If you are using common dependencies in several modules (e.g spring), it's better to move them to main build.gradle script under subproject {}
https://docs.gradle.org/current/userguide/declaring_dependencies_between_subprojects.html

Reply:
fixed

Reply:
Dependencies is still there